networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/Eth2NetworkBuilder.java

```diff
@@ -50,7 +54,10 @@
   private Eth2Config eth2Config;
   private EventBus eventBus;
   private RecentChainData recentChainData;
-  private GossipedAttestationConsumer gossipedAttestationConsumer;
+  private GossipedOperationConsumer<ValidateableAttestation> gossipedAttestationConsumer;
+  private GossipedOperationConsumer<AttesterSlashing> gossipedAttesterSlashingConsumer;
+  private GossipedOperationConsumer<ProposerSlashing> gossipedProposerSlashingConsumer;
+  private GossipedOperationConsumer<SignedVoluntaryExit> gossipedVoluntaryExitConsumer;
```

Comment:
We should either set these to a no-op value by default, or `validate()` that they get set if we want them to be required.  I'd think setting them with no-op values is probably fine.

Reply:
Validating makes sense to me.